<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item row="2" column="1">
        <widget class="QLineEdit" name="z_position">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Position Y</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="x_position">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Position X</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="y_position">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Position Z</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="1">
        <widget class="QLineEdit" name="q1"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>q0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="q0"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="q2"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>q1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="q3"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>q2</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>q3</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Vicon name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vicon_name"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Target X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="x_target"/>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Target Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="y_target"/>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Target Z</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="z_target"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="controller_start">
         <property name="text">
          <string>Start!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="emergency_recover">
         <property name="text">
          <string>Emergency Recover</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="emergency_stop">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Emergency Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="flip_start">
         <property name="text">
          <string>Flip!</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>218</width>
           <height>38</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="start_write">
           <property name="text">
            <string>start writing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="save_data">
           <property name="text">
            <string>save data</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
